### ISSUE TITLE
Check for merge conflict with awaiting-review

### ISSUE THREAD

**Issue by SimonLab** [dwyl/dwylbot#121]
When a user add the "awaiting-review" label, dwylbot will report an error if the PR has a merge conflict

**Comment by iteles** [comment]
A question on this, though it may be a case for a new issue:
https://github.com/dwyl/dwylbot/issues

Often it's the case that merging in of a different pull request causes an existing PR (already with an `awaiting-review` label) to have merge conflicts, despite the fact that it didn't when the PR was originally created.
e.g. https://github.com/dwyl/dwyl-site/pull/318

Could we/should we be accounting for this @markwilliamfirth ?

**Comment by ghost** [comment]
@iteles yes I've made a new issue: https://github.com/dwyl/dwylbot/issues/162
